src/pipelines/epidemiology/gb_authority.py

```diff
@@ -14,24 +14,65 @@
 
 from pathlib import Path
 from typing import Any, Dict, List
-from pandas import DataFrame, isnull, merge
+from pandas import DataFrame, concat
 from lib.pipeline import DataSource
 from lib.utils import table_rename
 from lib.time import datetime_isoformat
 from lib.cast import safe_int_cast
 import requests
+from uk_covid19 import Cov19API
+
+
+# Maps unitary authority (UTLA) to Local Health Board in Wales
+# source : https://data.gov.uk/dataset/0ae307c2-3092-41e7-a3d0-5d268c8d3662/unitary-authority-to-local-health-board-april-2019-lookup-in-wales
+wales_utla_map = {
+    "W06000001": "W11000023",
+    "W06000002": "W11000023",
+    "W06000003": "W11000023",
+    "W06000004": "W11000023",
+    "W06000005": "W11000023",
+    "W06000006": "W11000023",
+    "W06000023": "W11000024",
+    "W06000008": "W11000025",
+    "W06000009": "W11000025",
+    "W06000010": "W11000025",
+    "W06000018": "W11000028",
+    "W06000019": "W11000028",
+    "W06000020": "W11000028",
+    "W06000021": "W11000028",
+    "W06000022": "W11000028",
+    "W06000014": "W11000029",
+    "W06000015": "W11000029",
+    "W06000013": "W11000030",
+    "W06000016": "W11000030",
+    "W06000024": "W11000030",
+    "W06000011": "W11000031",
+    "W06000012": "W11000031",
+}
 
 
 class Covid19UkL2DataSource(DataSource):
-    def parse_dataframes(
-        self, dataframes: Dict[str, DataFrame], aux: Dict[str, DataFrame], **parse_opts
-    ) -> DataFrame:
+    def parse(self, sources: Dict[str, str], aux: Dict[str, DataFrame], **parse_opts) -> DataFrame:
+        cases_and_deaths = {
+            "date": "date",
+            "areaName": "areaName",
+            "newCasesByPublishDate": "newCasesByPublishDate",
+            "cumCasesByPublishDate": "cumCasesByPublishDate",
+            "newDeathsByPublishDate": "newDeathsByPublishDate",
+            "cumDeathsByPublishDate": "cumDeathsByPublishDate",
+            "cumPillarOneTestsByPublishDate": "cumPillarOneTestsByPublishDate",
```

Comment:
What is the purpose of this dict where keys = values?

Reply:
In theory, I am supposed to be able to rename the columns using this format, but it does not work as intended at the moment.